bot/exts/info/pypi.py

```diff
@@ -36,42 +37,49 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
-            await ctx.send(embed=embed)
-            return
 
-        async with self.bot.http_session.get(URL.format(package=package)) as response:
-            if response.status == 404:
-                embed.description = "Package could not be found."
+        else:
+            async with self.bot.http_session.get(URL.format(package=package)) as response:
+                if response.status == 404:
+                    embed.description = "Package could not be found."
 
-            elif response.status == 200 and response.content_type == "application/json":
-                response_json = await response.json()
-                info = response_json["info"]
+                elif response.status == 200 and response.content_type == "application/json":
+                    response_json = await response.json()
+                    info = response_json["info"]
 
-                embed.title = f"{info['name']} v{info['version']}"
-                embed.url = info['package_url']
-                embed.colour = next(PYPI_COLOURS)
+                    embed.title = f"{info['name']} v{info['version']}"
+                    embed.url = info['package_url']
+                    embed.colour = next(PYPI_COLOURS)
 
-                for field in FIELDS:
-                    field_data = info[field]
+                    for field in FIELDS:
+                        field_data = info[field]
 
-                    # Field could be completely empty, in some cases can be a string with whitespaces, or None.
-                    if field_data and not field_data.isspace():
-                        if '\n' in field_data and field == "license":
-                            field_data = field_data.split('\n')[0]
+                        # Field could be completely empty, in some cases can be a string with whitespaces, or None.
+                        if field_data and not field_data.isspace():
+                            if '\n' in field_data and field == "license":
+                                field_data = field_data.split('\n')[0]
 
-                        embed.add_field(
-                            name=field.replace("_", " ").title(),
-                            value=escape_markdown(field_data),
-                            inline=False,
-                        )
+                            embed.add_field(
+                                name=field.replace("_", " ").title(),
+                                value=escape_markdown(field_data),
+                                inline=False,
+                            )
 
-            else:
-                embed.description = "There was an error when fetching your PyPi package."
-                log.trace(f"Error when fetching PyPi package: {response.status}.")
+                    error = False
 
-        await ctx.send(embed=embed)
+                else:
+                    embed.description = "There was an error when fetching your PyPi package."
```

Comment:
I feel setting `error` To `False` as default is better and then setting it to `True` here.

But yeah that’s just my personal preference.